property_sets.gemspec

```diff
@@ -8,7 +8,7 @@ Gem::Specification.new "property_sets", PropertySets::VERSION do |s|
   s.homepage = 'http://github.com/zendesk/property_sets'
   s.license  = 'Apache License Version 2.0'
 
-  s.add_runtime_dependency("activerecord", ">= 4.2", "< 6.0")
+  s.add_runtime_dependency("activerecord", ">= 4.2")
```

Comment:
The question is whether we’ll allow people to use this gem with Rails 6.1 when it’s released, or whether we want to explicitly test compatibility first. I would consider adding a `< 6.1` restriction.